### PR TITLE
fix plugins display in list of routes

### LINKED ISSUE #2665: Plugins not shown on and not aligned in list of routes
### Issue description

1. The Plugin column is not aligned properly as shown in the first screenshot.
2. When the Operation column is fixed to the right, the Plugin column is not shown at all. See second screenshot.

### Expected behavior

1. The alignment should be proper like the other columns.
2. Plugins column should be shown even when the operations column is fixed to the right.

### How to Reproduce

Go to the routes page, create a route and see it listed in the page.

### Screenshots

1.

Screenshot 2022-11-11 at 11 37 07 AM

2.

Screenshot 2022-11-11 at 11 36 57 AM

### PR DESCRIPTION
**Pull request by incubator4**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
1. Add attribute `plugins` to `ResponseBody`
2. Add width attribute to columns of `labels` and `plugins` (Due to without width, these columns might not display  as expected)
3.  Add new render function for `plugins` columns.

**Related issues**

fix/resolve #2665 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

Why not test cases and corresponding document? Cause of it just was a bug fix.

### CODE REVIEW

**Comment by SkyeYoung** [comment]
Hi, could you add some comparison pictures to facilitate community members' review PR? (Because we don't have an online preview yet 😢)

**Comment by incubator4** [comment]
> Hi, could you add some comparison pictures to facilitate community members' review PR? (Because we don't have an online preview yet 😢)

My apologize for not replying soon. This following might show difference between fix bugs and before.

### Tags

cause of tags don't have attribute width, it might not be displayed correctly when the screen width is too short

#### Before

image

#### After

image


### Tags

due to plugins have neither width nor override render function. 

#### Before

image


#### After

image

**Review by Baoyuantop — APPROVED** [review]
Code LGTM, please make CI pass

File: web/src/pages/Route/List.tsx [hunk 1]
```diff
@@ -550,6 +551,13 @@ const Page: React.FC = () => {
     {
       title: formatMessage({ id: 'menu.plugin' }),
       dataIndex: 'plugins',
+      width: 240,
+      render: (_, record) => {
+        const plugins = record.plugins || {};
+        return Object.keys(plugins).length > 0
+          ? Object.keys(plugins).map((key) => <Tag key={key}> {key}</Tag>)
```
Comment (LiteSun) [inline web/src/pages/Route/List.tsx:558]:
```suggestion
          ? Object.keys(plugins).map((key) => <Tag key={key}>{key}</Tag>)
```

**Pull request merged**